src/common-components/PasswordField.jsx

```diff
@@ -29,9 +29,9 @@ const PasswordField = (props) => {
   const ShowButton = (
     <IconButton icon={faEye} onClick={setHiddenFalse} alt={formatMessage(messages['show.password'])} aria-label={formatMessage(messages['show.password'])} />
   );
-
+  const placement = window.innerWidth < 768 ? 'top' : 'left';
```

Comment:
instead of this can we do https://edx.github.io/paragon/components/responsive/ ?